package.json

```diff
@@ -24,45 +24,45 @@
   },
   "homepage": "https://github.com/decred/dcrdata#readme",
   "devDependencies": {
-    "@babel/core": "~7.4.0",
-    "@babel/plugin-syntax-dynamic-import": "~7.2.0",
-    "@babel/preset-env": "~7.4.2",
-    "axios": "~0.18.0",
-    "babel-eslint": "~10.0.1",
-    "babel-loader": "~8.0.5",
-    "babel-polyfill": "~6.26.0",
-    "clean-webpack-plugin": "~1.0.1",
-    "css-loader": "~1.0.1",
-    "eslint": "~5.16.0",
-    "eslint-config-standard": "~12.0.0",
-    "eslint-loader": "~2.1.2",
-    "eslint-plugin-import": "^2.17.2",
-    "eslint-plugin-node": "~8.0.1",
-    "eslint-plugin-promise": "~4.1.1",
+    "@babel/core": "^7.4.5",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/preset-env": "^7.4.5",
+    "axios": "^0.19.0",
```

Comment:
axios update appears to be non-breaking, but there are new features we could look into, such as `getURI`.